srtcore/sync.h

```diff
@@ -943,9 +943,74 @@ CUDTException& GetThreadLocalError();
 /// @param[in] maxVal maximum allowed value of the resulting random number.
 int genRandomInt(int minVal, int maxVal);
 
+class SharedMutex
+{
+    private:
+    Condition  m_pLockWriteCond;
+    Condition  m_pLockReadCond;
+
+    Mutex m_pMutex;
+
+    int m_pCountRead;
+    bool m_pWriterLocked;
```

Comment:
'p' means "pointer". This variable is of a type "Boolean", so should be `m_bWriterLocked`.